bin/pycbc_live

```diff
@@ -216,31 +222,47 @@ class LiveEventManager(object):
 
         return out
 
-    def get_followup_info(self, coinc_ifo, ifo, triggers, snr_series, ptime,
-                          pvalue, sigma2, recalculate_ifar=False):
-        # Copy the common fields from the other detector
+    def get_followup_info(
+        self,
+        coinc_ifo,
+        ifo,
+        triggers,
+        pvalue_info,
```

Comment:
this is reasonable, but to make the code more readable (eg  repetition of `pvalue_info['peak_time']`) I would unpack the dict entries immediately .. which presumably also makes the diff smaller